### ISSUE TITLE
Misaligned inventory in chests

### ISSUE THREAD

**Issue by SlothInTheBroth** [Reoseah/reinstorage#2]
![2019-06-02_11 20 07](https://user-images.githubusercontent.com/51274618/58764063-7040ed00-8528-11e9-95bc-45ba61294420.png)
Everything in my inventory is misaligned to the left. Also you increased the storage? You state that you made it configurable but I see no way to configure it, in both the config folder and in the Mod Menu.

**Comment by Reoseah** [comment]
it should create `reinstorage.txt` file in `config` dir on launch

and I found this displaying issue happens when there's 0 in config
```
additional-slots.gold=0
```
![2019-06-02_20 03 04](https://user-images.githubusercontent.com/20323084/58764585-8b841a80-8571-11e9-9884-d3a1e04a381a.png)
I'll probably ban actually putting zeroes in config.

in your case it's probably something with config file and I can't recreate that
try searching log for something like
```
[20:09:50] [main/ERROR]: Failed to load properties from file: .\config\reinstorage.txt
```
which it logs on read or write errors

**Comment by SlothInTheBroth** [comment]
You're correct, I'm sorry, there is a txt file in the config file. I had not changed anything in the config, and upon experimentation, nothing seems to change when I put a number, any number. The only exception is negative numbers, which crashes the game upon startup. In my log, nothing even included at least 'reinstorage', using the find function. I should add that this did not occur on the previous version, and that on the Double Reinforced Iron Chest, the two rows on the right appear, but do not function.

The reason this happened was because the chests were there before I updated. Destroying the chests and replacing them fixed this issue, albeit updating did permanently destroyed some of the items inside the chest. I appreciate your quick responses and overall helpfulness! Thank you!

This topic has been closed and solved.
